Author: <cajus@debian.org>
Description: Fixed location of auto-generated locales
--- a/include/php_setup.inc
+++ b/fusiondirectory-core/include/php_setup.inc
@@ -20,7 +20,7 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  */
 
-define ("LOCALE_DIR", dirname(dirname(__FILE__))."/locale/compiled");
+define ("LOCALE_DIR", "/var/cache/fusiondirectory/locale");
 
 function gosaRaiseError($errno, $errstr, $errfile, $errline)
 {
--- a/update-fusiondirectory
+++ b/fusiondirectory-core/update-fusiondirectory
@@ -169,20 +169,20 @@
 
 	/* For each language, merge the target .mo to the compiled directory. */
 	foreach ($languages as $language => $po_files){
-		if (!is_dir(LOCALE_DIR."/compiled/${language}/LC_MESSAGES")){
-			if (!mkdir (LOCALE_DIR."/compiled/${language}/LC_MESSAGES", 0755, TRUE)){
-				echo "Failed to create '".LOCALE_DIR."/compiled/${language}/LC_MESSAGES'- aborted";
+		if (!is_dir("/var/cache/fusiondirectory/locale/${language}/LC_MESSAGES")){
+			if (!mkdir ("/var/cache/fusiondirectory/locale/${language}/LC_MESSAGES", 0755, TRUE)){
+				echo "Failed to create '/var/cache/fusiondirectory/locale/${language}/LC_MESSAGES'- aborted";
 				exit (3);
 			}
 		}
 
 		/* Cat all these po files into one single file */
-		system ("(cd ".LOCALE_DIR." && msgcat --use-first ".implode(" ", $po_files)." > compiled/${language}/LC_MESSAGES/messages.po)", $val);
+		system ("(cd ".LOCALE_DIR." && msgcat --use-first ".implode(" ", $po_files)." > /var/cache/fusiondirectory/locale/${language}/LC_MESSAGES/messages.po)", $val);
 		if ($val != 0){
 			echo "Merging of message files failed - aborted";
 			exit (4);
 		}
-		system ("(cd ".LOCALE_DIR."/compiled/${language}/LC_MESSAGES && msgfmt -o messages.mo messages.po && rm messages.po)", $val);
+		system ("(cd /var/cache/fusiondirectory/locale/${language}/LC_MESSAGES && msgfmt -o messages.mo messages.po && rm messages.po)", $val);
 		if ($val != 0){
 			echo "Compiling of message files failed - aborted";
 			exit (5);
