Index: fusiondirectory/fusiondirectory-core/include/class_plugin.inc
===================================================================
--- fusiondirectory.orig/fusiondirectory-core/include/class_plugin.inc	2010-09-29 01:50:55.000000000 +0200
+++ fusiondirectory/fusiondirectory-core/include/class_plugin.inc	2010-09-29 01:52:15.000000000 +0200
@@ -1256,7 +1256,7 @@
               }
           }
 
-          if (check_command($command)){
+          if (check_command($command) || TRUE){
 
               @DEBUG (DEBUG_SHELL, __LINE__, __FUNCTION__, __FILE__,$command,"Execute");
               exec($command, $arr, $returnCode);
