Index: pkg/core/include/class_plugin.inc
===================================================================
--- pkg.orig/core/include/class_plugin.inc	2011-03-19 12:48:24.000000000 +0100
+++ pkg/core/include/class_plugin.inc	2011-03-19 12:51:27.000000000 +0100
@@ -1257,7 +1257,7 @@
               }
           }
 
-          if (check_command($command)){
+          if (check_command($command) || TRUE){
 
               @DEBUG (DEBUG_SHELL, __LINE__, __FUNCTION__, __FILE__,$command,"Execute");
               exec($command, $arr, $returnCode);
