Index: fusiondirectory/fusiondirectory-core/include/functions.inc
===================================================================
--- fusiondirectory.orig/fusiondirectory-core/include/functions.inc	2010-09-29 01:53:06.000000000 +0200
+++ fusiondirectory/fusiondirectory-core/include/functions.inc	2010-09-29 01:53:46.000000000 +0200
@@ -3062,7 +3062,7 @@
         $command= preg_replace("/%userPassword/", $password, $command);
         $command= preg_replace("/%dn/", $dn, $command);
 
-        if (check_command($command)){
+        if (check_command($command) || TRUE){
           @DEBUG (DEBUG_SHELL, __LINE__, __FUNCTION__, __FILE__, $command, "Execute");
           exec($command);
         } else {
