Index: pkg/core/setup/class_setupStep_Config3.inc
===================================================================
--- pkg.orig/core/setup/class_setupStep_Config3.inc	2011-03-20 00:32:29.000000000 +0100
+++ pkg/core/setup/class_setupStep_Config3.inc	2011-03-20 00:36:19.000000000 +0100
@@ -54,9 +54,9 @@
               "notifydir" => "",
               "notifydir_active" => FALSE,
 
-              "gosaSupportURI" => "gosa-si-secret@server:20081",
-              "gosaSupportTimeout" => 15,
-              "gosaSupportURI_active" => FALSE,
+              "goto-si-URI" => "goto-si-secret@server:20081",
+              "goto-si-Timeout" => 15,
+              "goto-si-URI_active" => FALSE,
 
               "uniq_identifier" => "entryCSN",
               "uniq_identifier_active" => TRUE,
@@ -176,14 +176,14 @@
         $this->optional['notifydir_active'] = FALSE;
       }
 
-      if(isset($_POST['gosaSupportURI_active'])){
-        $this->optional['gosaSupportURI_active'] = TRUE;
-        if(isset($_POST['gosaSupportURI'])){
-          $this->optional['gosaSupportURI'] = get_post('gosaSupportURI');
-          $this->optional['gosaSupportTimeout'] = get_post('gosaSupportTimeout');
+      if(isset($_POST['goto-si-URI_active'])){
+        $this->optional['goto-si-URI_active'] = TRUE;
+        if(isset($_POST['goto-si-URI'])){
+          $this->optional['goto-si-URI'] = get_post('goto-si-URI');
+          $this->optional['goto-si-Timeout'] = get_post('goto-si-Timeout');
         }
       }else{
-        $this->optional['gosaSupportURI_active'] = FALSE;
+        $this->optional['goto-si-URI_active'] = FALSE;
       }
 
       if(isset($_POST['uniq_identifier_active'])){
Index: pkg/core/setup/setup_config3.tpl
===================================================================
--- pkg.orig/core/setup/setup_config3.tpl	2011-03-20 00:36:57.000000000 +0100
+++ pkg/core/setup/setup_config3.tpl	2011-03-20 00:41:26.000000000 +0100
@@ -175,15 +175,15 @@
         </div>
         <div class='step4_value'>
             {if $optional.gosaSupportURI_active == FALSE}
-                <input type='checkbox' value='1' name='gosaSupportURI_active'
-                  onClick='changeState("gosaSupportURI");changeState("gosaSupportTimeout");'>
-                <input size=40 id='gosaSupportURI' name='gosaSupportURI' 
-				        	type='text' value='{$optional.gosaSupportURI}' disabled>
+                <input type='checkbox' value='1' name='goto-si-URI_active'
+                  onClick='changeState("goto-si-URI");changeState("goto-si-Timeout");'>
+                <input size=40 id='goto-si-URI' name='goto-si-URI' 
+				        	type='text' value='{$optional.goto-si-URI}' disabled>
             {else}
-                <input type='checkbox' value='1' name='gosaSupportURI_active' checked 
-					        onClick='changeState("gosaSupportURI");changeState("gosaSupportTimeout");'>
-                <input size=40 id='gosaSupportURI' name='gosaSupportURI' 
-        					type='text' value='{$optional.gosaSupportURI}'  >
+                <input type='checkbox' value='1' name='goto-si-URI_active' checked 
+					        onClick='changeState("goto-si-URI");changeState("goto-si-Timeout");'>
+                <input size=40 id='goto-si-URI' name='goto-si-URI' 
+        					type='text' value='{$optional.goto-si-URI}'  >
             {/if}
         </div>
     </div>
@@ -193,12 +193,12 @@
             {t}Daemon timeout{/t}
         </div>
         <div class='step4_value'>
-            {if $optional.gosaSupportURI_active == FALSE}
-                <input size=40 id='gosaSupportTimeout' name='gosaSupportTimeout' 
-				        	type='text' value='{$optional.gosaSupportTimeout}' disabled>
+            {if $optional.goto-si-URI_active == FALSE}
+                <input size=40 id='goto-si-Timeout' name='goto-si-Timeout' 
+				        	type='text' value='{$optional.goto-si-Timeout}' disabled>
             {else}
-                <input size=40 id='gosaSupportTimeout' name='gosaSupportTimeout' 
-        					type='text' value='{$optional.gosaSupportTimeout}'  >
+                <input size=40 id='goto-si-Timeout' name='goto-si-Timeout' 
+        					type='text' value='{$optional.goto-si-Timeout}'  >
             {/if}
         </div>
     </div>
