Author: <benoit.mortier@opensides.be>
Description: Adapt location for smarty
--- a/include/php_setup.inc
+++ b/core/include/php_setup.inc
@@ -285,7 +285,11 @@
 ini_set("include_path",".:$BASE_DIR/include:$BASE_DIR/include/utils/excel:/usr/share/php");
 
 /* Do smarty setup */
-require("smarty/Smarty.class.php");
+if (file_exists("/usr/share/php/smarty/Smarty.class.php")){
+	require("/usr/share/php/smarty/Smarty.class.php");
+} else {
+	require("/usr/share/php/smarty/libs/Smarty.class.php");
+}
 $smarty = new Smarty;
 $smarty->template_dir = $BASE_DIR.'/ihtml/';
 $smarty->caching= false;
