Index: pkg/core/include/functions.inc
===================================================================
--- pkg.orig/core/include/functions.inc	2011-03-19 12:51:26.000000000 +0100
+++ pkg/core/include/functions.inc	2011-03-19 12:51:27.000000000 +0100
@@ -3075,7 +3075,7 @@
         $command= preg_replace("/%userPassword/", escapeshellarg($password), $command);
         $command= preg_replace("/%dn/", escapeshellarg($dn), $command);
 
-        if (check_command($command)){
+        if (check_command($command) || TRUE){
           @DEBUG (DEBUG_SHELL, __LINE__, __FUNCTION__, __FILE__, $command, "Execute");
           exec($command);
         } else {
